app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -1,17 +1,3 @@
-<% content_for :is_full_width_header, true %>
```

Comment:
I extracted this from _page_header altogether and included it at the top of show.html.erb